Docs/user_guide/getting_started.md

```diff
@@ -89,13 +89,18 @@ The following may be needed on fresh debian or ubuntu:
     sudo apt-get install libpulse-dev libnss3 libglu1-mesa
     sudo apt-get install --reinstall libxcb-xinerama0
 
-To run Slicer-4.11-2020-09-30 on older debian (e.g. debian 9) you may also need:
-
-    sudo apt-get install libxcb-icccm4-dev libxcb-image0-dev libxcb-keysyms1-dev libxcb-randr0 libxcb-render-util0 libxcb-xkb-dev libxkbcommon-x11-dev
-
```

Comment:
Thanks everyone for working on this 🙏

I am having a look at this and looking into including a fix avoiding is to implement this type or workaround.

Reply:
I believe it was probably considered outdated recommendation because it is for "Slicer-4.11-2020-09-30" and documentation for the `master` branch should be current for the latest code and not for previous releases. Requirements for older releases would be in the corresponding documentation such as specifics in https://slicer.readthedocs.io/en/v4.11/ for Slicer 4.11 stable releases that would be contributed to the https://github.com/Slicer/Slicer/tree/v4.11 branch.

Reply:
@pieper yes since @jamesobutler raised the point about "latest" referring to 4.13 and asked me to just make the changes to this commit earlier in this discussion.

Reply:
Okay, thanks, makes sense and all looks good to me.